pkgs/by-name/py/pywebcopy/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

```suggestion
python3.pkgs.buildPythonApplication rec {
```

if this is a python package that can be imported then it should be added to python-modules